composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "artesaos/seotools",
+  "name": "cohensive/seotools",
```

Comment:
Don't change it

Reply:
Btw. you should make it on different branch than master.

Reply:
Right. Forgot I made the PR. Had to push new package while waiting for PR to be merged.